Themis/Views/Assessment/File Upload Exercise/FileErrorIcon.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  FileErrorIcon.swift
+//  Themis
+//
+//  Created by Tarlan Ismayilsoy on 02.10.23.
+//
+
+import SwiftUI
+
+/// Used for indicating file-related problems
+struct FileErrorIcon: View {
+    var backgroundColor = Color(UIColor.systemGray5)
+    
+    var body: some View {
+        Image(systemName: "doc.fill")
+            .font(.system(size: 80))
+            .padding(.bottom)
+            .overlay {
+                Image(systemName: "xmark")
+                    .font(.system(size: 30, weight: .bold))
+                    .padding(.top)
+                    .foregroundColor(backgroundColor)
+            }
+            .foregroundColor(.secondary)
+    }
+}
+
+#Preview {
+    FileErrorIcon()
+}
```

Comment:
iOS17 is not yet supported on the build agent where we build Themis. Therefore, we need to use the old syntax so the build succeeds.

```suggestion
struct FileErrorIcon_Previews: PreviewProvider {
     static var previews: some View {
         FileErrorIcon()
     }
}
```